config/initializers/25_carrierwave.rb

```diff
@@ -0,0 +1,12 @@
+CarrierWave.configure do |config|
+  config.fog_credentials = {
+    :provider => "Local",
+    :local_root => "#{Rails.root}/public",
+  }
+  config.fog_directory = "/system"
+
+  unless APP_CONFIG["fog"].nil?
+    config.fog_credentials = APP_CONFIG["fog"]["credentials"] unless APP_CONFIG["fog"]["credentials"].nil?
```

Comment:
We are running Ruby 2.3+ now, so we can use `Hash#dig` here. 

```suggestion
    config.fog_credentials = APP_CONFIG.dig("fog", "credentials") unless APP_CONFIG.dig("fog", "credentials").nil?
```